source/Nevermore/Advanced/SqlConnectionStringHelpers.cs

```diff
@@ -0,0 +1,80 @@
+using Microsoft.Data.SqlClient;
+using Octopus.TinyTypes;
+
+namespace Nevermore.Advanced
+{
+    public static class SqlConnectionStringHelpers
+    {
+        public static void OverrideValueIfNotSet(SqlConnectionStringBuilder connectionStringBuilder, DbConnectionStringKeyword propertyName, object overrideValue)
+        {
+            if (!connectionStringBuilder.ShouldSerialize(propertyName.Value))
+            {
+                connectionStringBuilder[propertyName.Value] = overrideValue;
+            }
+        }
+    }
+
+    public class DbConnectionStringKeyword : TinyType<string>
```

Comment:
👍 

Lets not add TinyTypes to Nevermore, we explicitly worked around referencing it. Instead just throw the `Value` property onto this type instead.